src/electionguard_cli/import_ballots/import_ballots_input_retrieval_step.py

```diff
@@ -63,12 +65,21 @@ def _get_encryption_devices(
 
     @staticmethod
     def _get_guardians_from_keys(
-        guardian_keys: str, context: CiphertextElectionContext
+        guardian_keys_dir: str, context: CiphertextElectionContext
     ) -> List[Guardian]:
-        guardian_private_records = from_list_in_file(
-            PrivateGuardianRecord, guardian_keys
-        )
+
+        files = listdir(guardian_keys_dir)
         return [
-            Guardian.from_private_record(g, context.number_of_guardians, context.quorum)
-            for g in guardian_private_records
+            ImportBallotsInputRetrievalStep._get_guardian(guardian_keys_dir, f, context)
+            for f in files
         ]
+
+    @staticmethod
+    def _get_guardian(
```

Comment:
This seems like this is actually just a method that should be getting the private records. This could also be pluralized to just get all the records .

```
def load_private_record(
          guardian_dir: str, filename: str
    ) -> PrivateGuardianRecord:
        full_file = join(guardian_dir, filename)
       return from_file(PrivateGuardianRecord, full_file)
```